proto/lavanet/lava/pairing/query.proto

```diff
@@ -235,6 +235,7 @@ message QueryEffectivePolicyResponse {
 message QuerySdkPairingResponse {
   QueryGetPairingResponse pairing = 1;
   uint64 max_cu = 2;
-  lavanet.lava.spec.Spec spec = 3; 
+  lavanet.lava.spec.Spec spec = 3;
+  int64 latest_block_time = 4;
```

Comment:
in the SDK we don't need the latest block time, as we query only once every epoch, meaning we can just check if an epoch has changed at the planned time, if not we can try again in a few seconds and if it remains for X% of the epoch duration we can trigger emergency mode. 

what you do need to add there are the downtime params.

Reply:
but how will we understand how many virtual epochs have already passed, without latest block time?